bears/generate_package.py

```diff
@@ -113,7 +117,13 @@ def main():
     else:
         bear_version = repr(bear_version) + '.' + str(int(time.time()))
 
-    for bear_file_name in sorted(set(glob('bears/**/*Bear.py'))):
+    if args.bear:
+        bear_files = [f for f in sorted(set(glob('bears/**/*Bear.py')))
+                      for x in args.bear if x in f]
+    else:
+        bear_files = sorted(set(glob('bears/**/*Bear.py')))
```

Comment:
You are using `sorted(set(glob('bears/**/*Beary.py')))` in both the `if` and `else` branches, why not store it in another variable and then use it.